src/components/accordion/index.md

```diff
@@ -54,10 +54,10 @@ Accordions, [tabs](/components/tabs/) and [details](/components/details/) all wo
 
 If you decide to use one of these components, consider if:
 
-- the user needs to look at more than one section at a time — an accordion can show multiple sections at a time, unlike tabs
-- the user needs to switch quickly between sections — tabs can show content without pushing other sections down the page, unlike accordions
-- there are many sections of content — accordions can fit more sections as they’re arranged vertically, unlike tabs which are arranged horizontally
-- there’s only one or two pieces of short, less important content — the details component is more suitable as it’s visually smaller and less prominent than an accordion or tabs
+- the user needs to look at more than one section at a time – an accordion can show multiple sections at a time, unlike tabs
```

Comment:
Because it's difficult to see what the difference is, just wanted to mention that this changes em-dashes to en-dashes. (This type of change could be its own commit to make that clearer.)